data/src/main/kotlin/com/vultisig/wallet/data/models/Chain.kt

```diff
@@ -209,4 +214,34 @@ val Chain.hasReaping: Boolean
     get() = when (this) {
         Chain.Polkadot, Chain.Ripple -> true
         else -> false
-    }
\ No newline at end of file
+    }
+
+val Chain.isBtcLike: Boolean
```

Comment:
you can check `chain.standard is TokenStandard.UTXO` thus don't need to add this

Reply:
thanks going to change it

Reply:
done